DEVELOP.rst

```diff
@@ -2,6 +2,32 @@
 Developer Guide
 ===============
 
+
+Building from source
+====================
+
+To build the CrateDB Prometheus Adapter from source, you need to have a working
+Go environment with **Golang version 1.16** installed.
+
+Use the ``go`` tool to download and install the ``cratedb-prometheus-adapter`` executable
+into your ``GOPATH``:
+
+.. code-block:: console
+
+   go get github.com/crate/cratedb-prometheus-adapter
+   cd $GOPATH/src/github.com/crate/cratedb-prometheus-adapter
```

Comment:
Can we explain this scenario a bit better? Why do I want to `go get` the `cratedb-prometheus-adapter` into my `$GOPATH`? Would this apply if I want to use it as a library?

Reply:
Good catch. This line is probably a copy-paste error.

Regarding `go get`: This command will download the sources, build and install the `cratedb-prometheus-adapter` executable into the `$GOPATH/bin` directory (which is usually part of your `$PATH` variable).

Reply:
All right, so just use `go get github.com/crate/cratedb-prometheus-adapter` here?

Reply:
Exactly :)

Reply:
Wonderful. Thank you again.